File: app/src/main/java/com/example/coolvideo/ui/Video/Danmu.java
package com.example.coolvideo.ui.Video;

public class Danmu {
    private Long time;
    private String content;

    public Danmu(Long time, String content) {
        this.time = time;
        this.content = content;
    }

    public Long getTime() {
        return time;
    }

    public String getContent() {
        return content;
    }
}

File: app/src/main/java/com/example/coolvideo/data/model/Danmu.kt
package com.example.coolvideo.data.model

import com.google.gson.annotations.SerializedName

data class Danmu (

    @SerializedName("id")
    var danmuId: Int,

    @SerializedName("userId")
    var userId: String,

    @SerializedName("videoId")
    var videoId: String,

    @SerializedName("time")
    var time: Long,

    @SerializedName("content")
    var content:String
)